### PR TITLE
fix header width & little current link changes

### PR DESCRIPTION
**Pull request by Yur1yBubl1k**

### CODE REVIEW

File: src/components/BurgerMenu/BurgerMenu.tsx
Comment (artem8746) [inline src/components/BurgerMenu/BurgerMenu.tsx:1]: replace `<a>` by `<Link>`, remove currentUrl and setCurrentUrl, delete `onClick` on links

Comment (artem8746) [reply src/components/BurgerMenu/BurgerMenu.tsx:1]: replace `ul` by `nav` and delete `li`

File: src/components/BurgerMenu/BurgerMenu.tsx [hunk 1]
```diff
@@ -100,7 +104,14 @@ export const BurgerMenu: React.FC<Props> = ({
...
           >
-            <img src="./logos/favourites.svg" alt="logoFavourite" />
+            { currentURL !== 'favourites' ? <img
+              src="./logos/favourites.svg"
+              alt="logoFavourite"
+            />
+              : <img
+                src="./logos/favourites-selected.svg"
+                alt="logoFavouriteSelected"
+              />}
```
Comment (artem8746) [inline src/components/BurgerMenu/BurgerMenu.tsx]: make it spans, add classes(check in buttons.scss) and change it when it's active

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -28,13 +29,19 @@ export const Header = () => {
             <ul className="nav_list">
               <li className="list_item">
```
Comment (artem8746) [inline src/components/Header/Header.tsx]: replace it by `<nav>`  and remove `<li>`

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -28,13 +29,19 @@ export const Header = () => {
...
                 <NavLink
+                  onClick={() => {
+                    setCurrentURL('/');
+                  }}
```
Comment (artem8746) [inline src/components/Header/Header.tsx]:
```suggestion

```
remove, you don't need onClick, when you use `to` attribute
change everywhere

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -62,13 +75,32 @@ export const Header = () => {
...
+              {currentURL !== 'favourites'
+                ? (
+                  <img src="./logos/favourites.svg" alt="logoFavourite" />
+                ) : (
+                  <img
+                    src="./logos/favourites-selected.svg"
+                    alt="logoFavourite"
+                  />
+                )
+              }
```
Comment (artem8746) [inline src/components/Header/Header.tsx]: use spans, learn (button.scss - `&-favourite`, and change classNames when status changed)

**Review by artem8746 — CHANGES_REQUESTED** [review]
💦💦💦💦💨💨💨💨
Amazing, just make a few changes

**Comment by artem8746** [comment]
remove scroll when burger menu opened

File: src/components/BurgerMenu/BurgerMenu.tsx [hunk 5]
```diff
@@ -1,38 +1,57 @@
...
 
-export const BurgerMenu: React.FC<Props> = ({ setCloseBurger }) => {
-  const [currentURL, setCurrentURL] = useState('');
+export const BurgerMenu: React.FC<Props> = ({
+  openBurger,
+  setCloseBurger,
+  currentURL,
+  setCurrentURL
+}) => {
+  // const [currentURL, setCurrentURL] = useState('');
```
Comment (sasholk) [inline src/components/BurgerMenu/BurgerMenu.tsx]: delete

File: src/components/Header/Header.tsx [hunk 6]
```diff
@@ -8,18 +10,33 @@ import { LogoLink } from '../LogoLink';
...
+
+//   if (normalizedQuery) {
+//     copyDevices = copyDevices.filter(
+//       device => device.title.toLowerCase().includes(normalizedQuery)
+//         || device.description.toLowerCase().includes(normalizedQuery),
+//     );
+//   }
+
+//   return copyDevices;
+// }
```
Comment (sasholk) [inline src/components/Header/Header.tsx]: delete

File: src/components/Header/Header.tsx [hunk 7]
```diff
@@ -8,18 +10,33 @@ import { LogoLink } from '../LogoLink';
...
 
 export const Header = () => {
+
   const [openBurger, setOpenBurger] = useState(false);
+  const [currentURL, setCurrentURL] = useState('');
+  const [sortField, setSortField] = useState('');
+  // const filteredDevices = filterDevices(qw, sortField);
```
Comment (sasholk) [inline src/components/Header/Header.tsx]: delete

**Review by sasholk — CHANGES_REQUESTED** [review]
good job!
but need to change namings `className` by BEM and also use nesting in scss

**Review by artem8746 — APPROVED** [review]
ヾ(^▽^*)))

**Pull request merged**